integration_tests/fixtures/activitiesApi/getActivity.json

```diff
@@ -2,6 +2,10 @@
   "id": 2,
   "prisonCode": "MDI",
   "attendanceRequired": true,
+  "inCell": false,
```

Comment:
This looks like a meaty JSON file to maintain. You happy with the fixtures method of returning stubbed API details?